src/view/components/decisionScopesComponent.jsx

```diff
@@ -0,0 +1,117 @@
+import "regenerator-runtime"; // needed for some of react-spectrum
+import React from "react";
+import { FieldArray } from "formik";
+import Button from "@react/react-spectrum/Button";
+import "@react/react-spectrum/Form"; // needed for spectrum form styles
+import PropTypes from "prop-types";
+import Textfield from "@react/react-spectrum/Textfield";
+import FieldLabel from "@react/react-spectrum/FieldLabel";
+import Delete from "@react/react-spectrum/Icon/Delete";
+import RadioGroup from "@react/react-spectrum/RadioGroup";
+import Radio from "@react/react-spectrum/Radio";
+import InfoTipLayout from "./infoTipLayout";
+import WrappedField from "./wrappedField";
+
+function DecisionScopesComponent({ values, formikProps, options }) {
```

Comment:
Please remove `formikProps` as a parameter.  The only time you use formikProps is to get values, but you already have values.

Reply:
thank you